manim/default.cfg

```diff
@@ -160,4 +160,5 @@ log_time = cyan dim
 log_message = 
 log_path = dim
 log_width = -1
-log_height = -1
\ No newline at end of file
+log_height = -1
+log_timestamps = True
```

Comment:
Question: It seems that there are no other files is this diff that actually make use of this new option. How/where is it being used?

Reply:
Oh wait, it's being converted to `log.timestamps`, right? That's confusing... but let's keep it for now.

Reply:
I agree that this is confusing, but it follows what the rest of the code does, so yeah :/